engine/runner/src/main/scala/org/enso/runner/Main.scala

```diff
@@ -307,7 +307,11 @@ object Main {
     */
   private def runRepl(projectPath: Option[String]): Unit = {
     val mainMethodName           = "internal_repl_entry_point___"
-    val dummySourceToTriggerRepl = s"$mainMethodName = Debug.breakpoint"
+    val dummySourceToTriggerRepl =
+      s"""from Builtins import all
```

Comment:
Yeah, it should be `Base` instead. Problem is, the repl tests (CI step "test REPL") are not using the distribution, so there's no stdlib. So I'm waiting till we have pure-enso tests running on CI and then we can switch.

Reply:
If I understand correctly, that will be done in #1046, right? In that case maybe it's worth adding a TODO here mentioning that issue, or adding this to the specification of that issue (so that I don't forget to add that :D).